convert2rhel/actions/conversion/preserve_only_rhel_kernel.py

```diff
@@ -50,34 +50,63 @@ def run(self):
             )
             return
 
-        # Check if kernel with same version is already installed.
+        # Check which of the kernel versions are already installed.
         # Example output from yum and dnf:
         #  "Package kernel-4.18.0-193.el8.x86_64 is already installed."
-        already_installed = re.search(r" (.*?)(?: is)? already installed", output, re.MULTILINE)
-        if already_installed:
```

Comment:
Dropping this condition, since it will always match something.
I can't imagine a case, where `yum install kernel` it won't return, that some kernel version is already installed at that point.